plugins/inputs/stackdriver/README.md

```diff
@@ -0,0 +1,100 @@
+# Stackdriver Input Plugin
+
+Stackdriver scrapes metrics from Google Cloud Monitoring.
+
+### Credentials
+
+This plugin uses a [Google Service Account](https://cloud.google.com/docs/authentication/getting-started) to interacting with the Stackdriver Monitoring API.
+
+### Configuration:
+
+```toml
+[[inputs.stackdriver]]
+  ## GCP Project
+  project = "projects/erudite-bloom-151019"
+  
+  ## API rate limit. On a default project, it seems that a single user can make
+  ## ~14 requests per second. This might be configurable. Each API request can
+  ## fetch every time series for a single metric type, though -- this is plenty
+  ## fast for scraping all the builtin metric types (and even a handful of
+  ## custom ones) every 60s.
+  # rateLimit = 14
+  
+  ## Every query to stackdriver queries for data points that have timestamp t
+  ## such that: (now() - lookbackSeconds) <= t < now(). Note that influx will
+  ## de-dedupe points that are pulled twice, so it's best to be safe here, just
+  ## in case it takes GCP awhile to get around to recording the data you seek.
+  # lookbackSeconds = 600
+  
+  ## Collection Delay Seconds (required - must account for metrics availability via Stackdriver Monitoring API)
+  # delaySeconds = 60
+  
+  ## Recommended: use metric 'interval' that equals to 'lookbackSeconds' to avoid
+  ## gaps or overlap in pulled data
+  interval = "10m"
```

Comment:
This means by default telegraf will only publish metrics from stackdriver every ten minutes, right? That's quite a long delay before data is available.

Reply:
The comment is updated.